lib/screens/images_to_pdf.dart

```diff
@@ -32,46 +36,71 @@ class _ImageToPdfState extends State<ImageToPdf> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      floatingActionButton: Row(
-        mainAxisAlignment: MainAxisAlignment.end,
-        children: [
-          if (_images.isNotEmpty)
-            Padding(
-              padding: const EdgeInsets.symmetric(horizontal: 4),
-              child: FloatingActionButton.extended(
-                heroTag: 'export',
-                onPressed: () => exportPdf(_images),
-                icon: const Icon(Icons.upload_file),
-                label: const Text('Export'),
-                backgroundColor: Colors.green,
-              ),
+        floatingActionButton: Padding(
```

Comment:
adapt with latest one .What you have done is ok but its outdated now.

Reply:
latest one as in?

Reply:
@nayakastha compare with older version here